File: solutions/r/raindrops/1/raindrops.R
raindrops <- function(number) {
  result <- ""
  if(number%%3==0) {
    result <- paste(result,"Pling",sep="")
  }
  if(number%%5==0) {
    result <- paste(result,"Plang",sep="")
  }
  if(number%%7==0) {
    result <- paste(result,"Plong",sep="")
  }
  else if(number%%5!=0&&number%%3!=0) {
    result <- sprintf("%d", number)
  }
  return(result)
}

File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
    bein = ""
    if number%3 == 0 bein = bein * "Pling" end
    if number%5 == 0 bein = bein * "Plang" end
    if number%7 == 0 bein = bein * "Plong" end
    bein == "" && return "$number"
    return bein
end
